### PR TITLE
Add example for Retreving Array Elements

### LINKED ISSUE #24748: Arrays
[it would me more useful if you included examples for retrieving array elements !]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8f3885e9-4e4e-897c-0324-2e8f93d217b7
* Version Independent ID: f9014ce9-93c1-cda7-6286-c4226150fce4
* Content: [Single-Dimensional Arrays - C# Programming Guide](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/arrays/single-dimensional-arrays)
* Content Source: [docs/csharp/programming-guide/arrays/single-dimensional-arrays.md](https://github.com/dotnet/docs/blob/main/docs/csharp/programming-guide/arrays/single-dimensional-arrays.md)
* Product: **dotnet-csharp**
* Technology: **csharp-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by Viniciusap**
## Summary

Small example of retriving data from arrays

Fixes #24748

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=24993) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=24993) before we can accept your contribution.<br/><br/>:x: Viniciusap [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=24993)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=24993) it.</sub>

File: docs/csharp/programming-guide/arrays/snippets/RetrevingArrayElements.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            Wed
+            Thu
+            Fri
+            Sat
+            */
+
+            //The code below will throw a IndexOutOfRangeException
+            //because is trying to invoke a not existent index
+
+            //Console.WriteLine(weekDays2[7]);
```
Comment (BillWagner) [inline docs/csharp/programming-guide/arrays/snippets/RetrevingArrayElements.cs:40]: I suggest moving these three lines about the displayed output.

File: docs/csharp/programming-guide/arrays/single-dimensional-arrays.md [hunk 2]
```diff
@@ -44,6 +44,12 @@ Consider the following array declaration:
 :::code language="csharp" source="snippets/SingleDimensionArrays.cs" id="FinalInstantiation":::
 
 The result of this statement depends on whether `SomeType` is a value type or a reference type. If it's a value type, the statement creates an array of 10 elements, each of which has the type `SomeType`. If `SomeType` is a reference type, the statement creates an array of 10 elements, each of which is initialized to a null reference. In both instances, the elements are initialized to the default value for the element type. For more information about value types and reference types, see [Value types](../../language-reference/builtin-types/value-types.md) and [Reference types](../../language-reference/keywords/reference-types.md).
+
+## Retreving data from Array
+
+You can retrieve the data of an array by use an index. For example:
+
+:::code language="csharp" source="snippets/RetrevingArrayElements.cs" id="RetrevingDataArray":::
```
Comment (BillWagner) [inline docs/csharp/programming-guide/arrays/single-dimensional-arrays.md:52]: We should be able to run this interactively with the following syntax:

```suggestion
:::code language="csharp" source="snippets/RetrevingArrayElements.cs" id="RetrevingDataArray" interactive="try-dotnet-method":::
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for creating this PR @Viniciusap 

It's a great improvement. I had a couple small suggestions. Once you accept those, this is ready to merge.

Thanks again for contributing to docs! 🎆

**Comment by Viniciusap** [comment]
Hi @BillWagner 
I think my last commit resolved what you ask me to do.
Thank you.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @Viniciusap 

I'll :shipit: now.

**Pull request merged**